document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -79,7 +83,15 @@ public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequ
             }
         }
 
-        return null;
+        try {
+            return smallFullAccountsDataHandler.getSmallFullAccountsData(transaction, resourceLink, requestId);
+        } catch (HandlerException e) {
+            LOG.errorContext(requestId,"An error occurred when calling the account handler to obtain " +
```

Comment:
We could define a constant for this error message and we could use `String.format` to customize it. 

```
ERROR_CALLING_ACCOUNTS = 
    "An error occurred when calling the account handler to obtain " +
    "abridged accounts data for transaction: %s  and resource link %s);
      
String.format(ERROR_CALLING_ACCOUNTS, transaction.getId(), transaction.getId())
```

Reply:
done, but without the function as not necessary as only used once now